### DISCUSSION TITLE
Search meeting priority and order + distance display

### DISCUSSION THREAD

**Discussion by nataliefaith2** [code4recovery/12-step-meeting-list#1255] [Q&A] [answered]
The new al-anon.org.au website is now live and meetings are being found and used by members.

The team have provided the below feedback to me - can you help me solve this issue?
"_There seems to be an issue with the meeting search function – especially the anywhere – it’s not showing the choice of kilometres like it was. No meetings come up when you put in near a location like Melbourne either, hopefully we can rectify this. It was showing my closest meeting but now it shows meetings that are on soon but not physically near me and doesn’t show the distance like I think it did._"

I recently changed the timezone to Melbourne instead of UTC, and I updated PHP to 8.0. The only other update that's been made is the team wanted the overarching regions to be visible instead of the sub-regions (like the town name). 

Would any of these updates effected the meeting display and caused these issues? 

I look forward to hearing your solution - much appreciated, Natalie

**Comment by joshreisner** [comment]
hi, there does appear to be an issue with the "Near Location" search, if i submit a location then it clears the field and switches to regular keyword search mode, so i've [created an issue for this](https://github.com/code4recovery/tsml-ui/issues/342). it's nothing you did, it is affecting all TSML UI sites. i'm surprised no one else reported it!

**Reply by joshreisner** [reply]
ok, that appears to have fixed it. you might need to refresh your page

**Reply by nataliefaith2** [reply]
I'm not see any changes - Do you think caching maybe stopping me from seeing this update?

**Reply by joshreisner** [reply, marked as answer]
it's working for me on your site now, maybe just refresh your browser? im able to change to 'near location' mode and when i search i see distance in kms

here is what i see:

![IMG_0066](https://github.com/code4recovery/12-step-meeting-list/assets/1551689/5ebaf6b0-f5a4-47d7-8495-4d32ee0dec99)

let me know if youre seeing something different
